src/app/pages/loading/loading.module.ts

```diff
@@ -1,20 +1,24 @@
-import {NgModule} from '@angular/core';
-import {CommonModule} from '@angular/common';
-import {FormsModule} from '@angular/forms';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
 
-import {IonicModule} from '@ionic/angular';
+import { IonicModule } from '@ionic/angular';
+import { RouterModule } from '@angular/router';
 
-import {LoadingPageRoutingModule} from './loading-routing.module';
-
-import {LoadingPage} from './loading.page';
+import { LoadingPage } from './loading.page';
 
 @NgModule({
   imports: [
     CommonModule,
     FormsModule,
     IonicModule,
-    LoadingPageRoutingModule
+    RouterModule.forChild([
+      {
+        path: '',
+        component: LoadingPage,
+      },
+    ]),
```

Comment:
What is this? Doesn't this belong in the router module?

Reply:
@wbratz ^^

Reply:
Oh, maybe this was probably just me trying to figure out why I couldn't navigate, I'll address.

Reply:
reverted changes, tested, everything still works.